packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -285,7 +285,7 @@ export class List extends BaseComponent<IListProps, IListState> {
   }
 
   public render() {
-    let { className } = this.props;
+    let { className, id, style } = this.props;
```

Comment:
you could use the getProperties(this.props, divProperties) to extract div props, then mix that in.

```
let divProps = getProperties(this.props, divProperties);

<div ref='root' { ...divProps } className={ ... } >
```

That way if you pass in other props, like aria- or data- or whatever, it will work. You don't need to only support id and style.